Api/KanbanizeApiV2.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Chaplean\Bundle\KanbanizeApiClientBundle\Api;
+
+use Chaplean\Bundle\ApiClientBundle\Api\AbstractApi;
+use Chaplean\Bundle\ApiClientBundle\Api\Parameter;
+use Chaplean\Bundle\ApiClientBundle\Api\Route;
+use GuzzleHttp\ClientInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class KanbanizeApiV2.
```

Comment:
Name is not good too. My proposals are KanbanizeApi or KanbanizeCurrentApi or KanbanizeNewApi or KanbanizeV2Api if you really want to put the version in the name